content-elements/form/poll/prototype/poll.js

```diff
@@ -20,14 +20,14 @@ Alpine.data('formPoll', () => ({
     range.setAttribute('class', 'bsi-poll-number-input');
 
     let container = this.root.querySelector('.bsi-poll-radio-group');
-
+    let name = "bsi-poll-radios-" +  Math.floor(Math.random() * (100 - 1 + 1)) + 1;
```

Comment:
Is that extra computation needed?
I'd suggest the name to be the same value as the ID property.
Since ID has to be unique, we don't need to compute random numbers.

Reply:
It is needed for multiple elements. E.g. We have 2 polls, and you don't validate the second one than even the first one will be red.